tests/api_tests/test_questionnaire_response_api.py

```diff
@@ -1707,6 +1707,70 @@ def test_questionnaire_life_functioning_survey(self):
         self.assertEqual(summary['questionnaireOnLifeFunctioningAuthored'], '2022-09-06T14:32:28')
         self.assertEqual(summary['questionnaireOnLifeFunctioningTime'], '2022-09-07T01:02:03')
 
+    def test_remote_identity_verified(self):
+        """ Test to see if a remote ID verification True Response saves successfully """
+        # Set up user config to have client set to vibrent
+        user_info = config.getSettingJson(config.USER_INFO)
+        original_user_client_id = user_info['example@example.com']['clientId']
+        user_info['example@example.com']['clientId'] = 'vibrent'
+        config.override_setting(config.USER_INFO, user_info)
+        # Set up participant, questionnaire, questionnaire response & send POST request to API
+        participant = self.data_generator.create_database_participant(participantOrigin='vibrent')
+        participant_id = f'P{participant.participantId}'
+        authored = datetime.datetime.now()
+        self.send_consent(participant.participantId, authored=authored)
+        questionnaire_id = self.create_questionnaire("remote_id_verification_questionnaire.json")
+        resource = self._load_response_json("remote_id_verification_questionnaire_response.json",
+                                            questionnaire_id,
+                                            participant_id)
+        self._save_codes(resource)
+        response = self.send_post(_questionnaire_response_url(participant_id), resource)
+        # Check the questionnaire response table to make sure data is accurate
+        response_obj = self.session.query(QuestionnaireResponse).filter(
+            QuestionnaireResponse.questionnaireResponseId == response['id']
+        ).one()
+        self.assertEqual(response_obj.answers[0].valueDecimal, 1)
+        self.assertEqual(response_obj.answers[1].valueDate, datetime.date(2022, 11, 30))
+        # Check the DAO to make sure data is accurate
+        qr = self.dao.get_with_children(response['id'])
+        self.assertEqual(qr.answers[0].valueDecimal, 1)
+        self.assertEqual(qr.answers[1].valueDate, datetime.date(2022, 11, 30))
+        # Reset Config back to original user client ID
+        user_info['example@example.com']['clientId'] = original_user_client_id
+        config.override_setting(config.USER_INFO, user_info)
+
+    def test_remote_identity_not_verified(self):
+        """ Test to see if a remote ID verification False Response saves successfully """
+        # Set up user config to have client set to vibrent
+        user_info = config.getSettingJson(config.USER_INFO)
+        original_user_client_id = user_info['example@example.com']['clientId']
+        user_info['example@example.com']['clientId'] = 'vibrent'
+        config.override_setting(config.USER_INFO, user_info)
```

Comment:
Shouldn't need this for the tests, you can do the POST with the default account.